File: src/main/java/com/example/demo/Entity/AirportMaster.java
package com.example.demo.Entity;

import jakarta.persistence.CascadeType;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "airport_master")
@Data
public class AirportMaster {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "airport_id")
    private int airportId;
	
	@Column(name = "airport_name")
	private String airportName;
	
	@ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "city_id")
    private CityMaster city;
	
    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "state_id")
    private StateMaster state;
    
    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "hub_id")
    private HubMaster hub;
    
    public int getAirportId() {
		return airportId;
	}

	public void setAirportId(int airportId) {
		this.airportId = airportId;
	}

	public String getAirportName() {
		return airportName;
	}

	public void setAirportName(String airportName) {
		this.airportName = airportName;
	}

	public CityMaster getCity() {
		return city;
	}

	public void setCity(CityMaster city) {
		this.city = city;
	}

	public StateMaster getState() {
		return state;
	}

	public void setState(StateMaster state) {
		this.state = state;
	}

	public HubMaster getHub() {
		return hub;
	}

	public void setHub(HubMaster hub) {
		this.hub = hub;
	}

	public String getAirportCode() {
		return airportCode;
	}

	public void setAirportCode(String airportCode) {
		this.airportCode = airportCode;
	}

	@Column(name = "airport_code")
    private String airportCode;

}

File: Backend.NET/Models/AirportMaster.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FleetManagementSystem.Api.Models;

[Table("airport_master")]
public class AirportMaster
{
    [Key]
    [Column("airport_id")]
    public int AirportId { get; set; }

    [Column("airport_name")]
    public string AirportName { get; set; }

    [Column("airport_code")]
    public string AirportCode { get; set; }

    [ForeignKey("City")]
    [Column("city_id")]
    public int? CityId { get; set; }
    public CityMaster City { get; set; }

    [ForeignKey("State")]
    [Column("state_id")]
    public int? StateId { get; set; }
    public StateMaster State { get; set; }

    [ForeignKey("Hub")]
    [Column("hub_id")]
    public int? HubId { get; set; }
    public HubMaster Hub { get; set; }
}
